Sources/Plasma/PubUtilLib/plAvatar/plAvTaskSeek.cpp

```diff
@@ -379,6 +380,27 @@ bool plAvTaskSeek::IMoveTowardsGoal(plArmatureMod *avatar, plAvBrainHuman *brain
         // We just set the pos/rot, so we know these are hit.
         fPosGoalHit = true;
         fRotGoalHit = true;
+    } else if (fRotGoalHit && fDistance <= .5f) {
```

Comment:
I've totally forgotten what units plasma uses. Feet? So this is ~6 inches of maximum warp?

Reply:
That's correct. I don't remember how much experimenting I did to come up with this magic number, but it works in my tests. For reference, [this](https://www.youtube.com/watch?v=Hj6SoWpejfA) is the issue I'm trying to fix.